CHANGELOG.md

```diff
@@ -3,4 +3,7 @@
 * Add Python 3.10 support (#133)
 * Add support for pre-populated Sessions (#132)
 * Add scaling policy endpoint (#136)
+* Drop Python 3.5 support
+* Up `requests` lib version 
```

Comment:
Did this happen? I’m not seeing it here.

Reply:
It was in other PR